ONSdigital/ssdc-rm-support-tool
# Add fulfilment trigger date time changer

# Motivation and Context

We want to be able to configure the fulfilment trigger date time.
# What has changed



A button has been added to the landing page that allows you to select a date time for the fulfilment trigger.
# How to test?



`make build` from the repo root to build everything.
Fire it up and go to the landing page. 
Click the `CONFIGURE FULFILMENT TRIGGER` button and change the date and time using the time picker.
Look in your database at the `fulfilment_next_trigger` table and check the date time matches what you put in.
# Links



https://trello.com/c/FzHwncKK/2579-support-tool-to-be-able-to-configure-fulfilment-batch-trigger-date-time-5
# Screenshots (if appropriate):
